### PR TITLE
Fix bug in admin v1 event modal

### PR DESCRIPTION
**Pull request by malaikaaiyar**
discussed in https://doesitsparkle.slack.com/archives/C01H9D1P1PE/p1628154814036200

makes the duration minutes field not cause errors

### CODE REVIEW

**Review by yurii-lubynets — APPROVED** [review]
LGTM! The optional field should be a good solution. Please don't merge until one more approve

**Comment by malaikaaiyar** [comment]
I tested it in both admin v1 and admin v3 and everything worked as expected

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx [hunk 1]
```diff
@@ -73,7 +73,9 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
         description: data.description,
         start_utc_seconds:
           start.unix() || Math.floor(new Date().getTime() / 1000),
-        duration_minutes: data.duration_hours * 60 + data.duration_minutes,
+        duration_minutes: data.duration_minutes
+          ? data.duration_hours * 60 + data.duration_minutes
+          : data.duration_hours * 60,
```
Comment (sunny-viktoryia) [inline src/components/organisms/TimingEventModal/TimingEventModal.tsx:78]: maybe this way would look better? what do you think?
```suggestion
        duration_minutes: data.duration_hours * 60 + (data.duration_minutes ?? 0)
```

**Review by sunny-viktoryia — APPROVED** [review]
Awesome! 😍 thanks for the fix!

there is a suggestion which i believe should improve readability a bit. :)

File: src/components/organisms/TimingEventModal/TimingEventModal.tsx (hunk 1 above)
Comment (malaikaaiyar) [reply src/components/organisms/TimingEventModal/TimingEventModal.tsx:78]: yeah i think that's better! i'll commit it

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**